### PR TITLE
Trg bit function and era-dependent template for wrem::goodMuonTriggerCandidate

### PR DESCRIPTION
**Pull request by ftenchini**
Testing ongoing.

So far: Tested for mw 2016 and 2018.

### CODE REVIEW

File: wremnants/muon_selections.py [hunk 1]
```diff
@@ -97,8 +97,8 @@ def select_z_candidate(df, mass_min=60, mass_max=120):
 
     return df
 
-def apply_triggermatching_muon(df, dataset, muon_eta, muon_phi, otherMuon_eta=None, otherMuon_phi=None, era = "2016PostVFP"):
-    df = df.Define("goodTrigObjs", f"wrem::goodMuonTriggerCandidate(TrigObj_id,TrigObj_filterBits,\"{era}\")")
+def apply_triggermatching_muon(df, dataset, muon_eta, muon_phi, otherMuon_eta=None, otherMuon_phi=None, era = "2016PostVFP"):≈ß
```
Comment (cippy) [inline wremnants/muon_selections.py]: Is this a typo? I see a  "~beta" after the colon

**Review by cippy — COMMENTED** [review]
I only had a minor comment, otherwise it looks fine. Let's wait for Suvankar's advice and that you complete your checks, then I think it can be merged

File: wremnants/muon_selections.py (hunk 1 above)
Comment (ftenchini) [reply wremnants/muon_selections.py]: That's the weirdest thing, that character is not there in my editor. Maybe something weird happened in the text formatting, let me check.

Comment (ftenchini) [reply wremnants/muon_selections.py]: That change only existed on the remote branch and appeared out of nowhere. 
I'm slightly baffled but anyway, I fixed it.

**Comment by sroychow** [comment]
Hi @ftenchini looks good to me as well. Have the tests finished? Then this can be merged.

**Review by sroychow — APPROVED** [review]

File: wremnants/muon_selections.py (hunk 1 above)
Comment (cippy) [reply wremnants/muon_selections.py]: Could be similar to what happens with thunderbird from a Mac OS system, sometimes white spaces or special characters appear formatted differently for those who receive the messages if they use a different operating system. 
Anyway, now that it is fixed I think this is ready for merge, but we can still wait for Suvankar for the last word

**Comment by cippy** [comment]
Ahah, Suvankar replied just when I wrote the other message, let's merge then :)

**Comment by sroychow** [comment]
Merging.

**Pull request merged**